.travis.yml

```diff
@@ -3,7 +3,8 @@ language: python
 matrix:
     include:
         - python: "3.6"
-          env: TOXENV=flake8
+          env: TOXENV=lint
+          after_success:
         - python: "2.7"
```

Comment:
Did this line slip in?

Reply:
This was intentional.

Reply:
Could you give me a hint about the reasoning?

Reply:
If you look at what `after_success` does it should be clear that it doesn't make sense for the linting step. The linting step produces no coverage data and coveralls will misinterpret that as "zero coverage".